packages/fxa-shared/db/index.ts

```diff
@@ -72,37 +72,37 @@ export function monitorKnexConnectionPool(
       numPendingAcquires: pool.numPendingAcquires(),
       // how many asynchronous create calls are running
       numPendingCreates: pool.numPendingCreates(),
-    };
+    });
   };
   pool.on('acquireRequest', (eventId: any) => {
-    log?.info('db.FXA-4648', {
-      msg: `Knex acquireRequest`,
+    log?.info('db', {
+      msg: `db.FXA-4648: Knex acquireRequest ${JSON.stringify(getPoolStats())}`,
```

Comment:
Won't this stringify a stringify?

Reply:
Good point. I'll move this into the poolstats field for consistency.